### PR TITLE
Updates and bug fixes

### PR DESCRIPTION
**Pull request by bhgrant8**
* **Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] Included your name and github handle in the contributors.md - Contributors section
- [x] Major/Breaking Changes are listed in Changelog

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Provides a functioning version of this template with some important updates for building a usable project.

  - Initial Commit
  - Basic README, cookiecutter.json, license, other docs added
  - Requirements file
  - Tox based testing
- Bug Fixes
- Moved from rst based to Markdown based documentation (Based on Hack Oregon policy)
- Example testing with pytest. Removed tox from project as adds unneeded complexity

* **What is the current behavior?** (You can also link to an open issue here)
Current master branch does not produce a functioning project, uses rst instead of markdown

Additionally these issues:

https://github.com/hackoregon/2019-backend-cookiecutter-django/issues/4
https://github.com/hackoregon/2019-backend-cookiecutter-django/issues/1
https://github.com/hackoregon/2019-backend-cookiecutter-django/issues/2

* **What is the new behavior (if this is a feature change)?**

This PR will address the fixes suggested for referenced issues. Additionally moves project to markdown based text, provides a PDF user guide for teams to setup projects, and adds contribution guide in addition to other fixes.

PR also provides a usable docker image and files for development work

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

This may break early projects who used the template. Teams should reach out to #team-backend in slack for hack oregon if there are concerns/issues

* **Other information**:

### CODE REVIEW

**Review by geophpherie — COMMENTED** [review]
I think @znmeb already created the project for transpo, but I'm not sure where he pulled it from. I'll let him chime in for us.

**Review by znmeb — APPROVED** [review]
Looks good to me. Yes, I've already built the `2019-transportation-systems-backend` but I'll merge all these changes into it after you merge to master.

**Pull request merged**
